activesupport/test/core_ext/range_ext_test.rb

```diff
@@ -193,12 +193,14 @@ def test_step_on_time_with_zone
 
   def test_include_on_time_with_zone
     twz = ActiveSupport::TimeWithZone.new(nil, ActiveSupport::TimeZone["Eastern Time (US & Canada)"], Time.utc(2006, 11, 28, 10, 30))
-    assert ((twz - 1.hour)..twz).include?(twz)
+    assert ((twz - 1.hour)..twz).cover?(twz)
   end
 
-  def test_case_equals_on_time_with_zone
+  def test_equals_on_time_with_zone
     twz = ActiveSupport::TimeWithZone.new(nil, ActiveSupport::TimeZone["Eastern Time (US & Canada)"], Time.utc(2006, 11, 28, 10, 30))
-    assert ((twz - 1.hour)..twz) === twz
+    assert_raises TypeError do
+      ((twz - 1.hour)..twz) === twz
+    end
```

Comment:
The actual error is `TypeError - can't iterate from ActiveSupport::TimeWithZone`. This test is making an assertion similar to how the `test_each_on_time_with_zone` and `test_step_on_time_with_zone` tests are wrriten above.

Reply:
This test is green in local but failing on CI with 

```
RangeTest#test_equals_on_time_with_zone [/rails/activesupport/test/core_ext/range_ext_test.rb:208]:
--
  | TypeError expected but nothing was raised.
```

Reverting it to the way it was earlier.